specs/beacon-chain.md

```diff
@@ -242,8 +236,8 @@ A `CrosslinkRecord` has the following fields:
 
 ```python
 {
-    # Dynasty number
-    'dynasty': 'int64',
+    # Since last validator set change?
+    'recently_changed': 'int8',
```

Comment:
Somewhat more semantic/readable as type `bool`

Reply:
bool is not currently supported in ssz. If we add it here, need to add it to the ssz spec

Reply:
(which I support)